sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -46,6 +46,18 @@ public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential) : thi
 
         /// <summary> The constitution is a script that assesses and applies proposals from consortium members. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GetConstitutionAsync.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new ConfidentialLedgerClient(endpoint, credential);
```

Comment:
Please add a carriage return after this line.

Reply:
done.